Database/QuestieDB.lua

```diff
@@ -832,39 +811,26 @@ function QuestieDB:GetQuest(questId) -- /dump QuestieDB:GetQuest(867)
 
     QO.SpecialObjectives = {}
 
-    local requiredSourceItems = rawdata[questKeys.requiredSourceItems]
-    if requiredSourceItems ~= nil then --required source items
+    local requiredSourceItems = QO.requiredSourceItems
+    if requiredSourceItems then --required source items
         for _, itemId in pairs(requiredSourceItems) do
-            if itemId ~= nil then
-                QO.SpecialObjectives[itemId] = {
-                    Type = "item",
-                    Id = itemId,
-                    Description = QuestieDB.QueryItemSingle(itemId, "name")
-                }
-            end
-        end
-    end
-
-    local zoneOrSort = rawdata[questKeys.zoneOrSort]
-    if zoneOrSort and zoneOrSort ~= 0 then
-        if zoneOrSort > 0 then
-            QO.Zone = zoneOrSort
-        else
-            QO.Sort = -zoneOrSort
```

Comment:
I guess QO.Sort was never used so you removed it?

Reply:
Both `.Zone` and `.Sort` are never used